src/utils.jl

```diff
@@ -231,8 +231,10 @@ end
 
 
 "Test for function definition expressions."
-isdef(ex) = isshortdef(ex) || longdef1(ex) !== nothing
+isdef(ex) = isshortdef(ex) || islongdef(ex)
 
+islongdef(ex)  = @capture(ex, function (fcall_ | fcall_) body_ end)
```

Comment:
This way of checking if an expression is a long function definition is taken from here:
[here](https://github.com/FluxML/MacroTools.jl/blob/master/src/utils.jl#L298-L300)

However, it seems to me that the or statement has no effect and the following would be equivalent:
```julia
islongdef(ex)  = @capture(ex, function fcall_ body_ end)
```

Reply:
I also wonder about the `|`... odd...

Independently of that, maybe also change https://github.com/FluxML/MacroTools.jl/blob/master/src/utils.jl#L298-L300 to use `islongdef`?

Reply:
Good catch, indeed, https://github.com/FluxML/MacroTools.jl/commit/69f5811d984e61df016acc085e7881d022ff8a8e shows that I'm the one who changed the more complex `|` expression to this `(fcall_ | fcall_)`. I'm going to guess that I did that purely because this is complex code and I wanted to play it safe. 

Long story short: please try it without the `|`, and if all tests pass, we'll keep it like that.